etc/input.conf

```diff
@@ -181,6 +181,38 @@
 #g-b script-binding select/select-binding
 #g-r script-binding select/show-properties
 
+#Alt+KP1 add video-rotate -1 # rotate video counterclockwise by 1 degree
+#Alt+KP5 set video-rotate  0 # reset rotation
+#Alt+KP3 add video-rotate  1 # rotate video clockwise by 1 degree
+
+#KP1 add video-zoom    -0.01 # zoom out video
```

Comment:
I found `0.01` good granularity. Especially if you hold the key it is more smooth to zoom in, but still is not too small step, imho.

I don't want to touch old keybinds to avoid endless discussion about the default value changes.